mystreams.py

```diff
@@ -320,7 +324,7 @@ def __init__(
         show_fps: bool = False,
         stream_depth: int = 10,
         allow_drop: bool = False,
-        multiplex = False,
+        multiplex=False,
```

Comment:
Hmm, tonly this formatting gets changed. Did you format the file with Black as we do?

Reply:
yes, it matches the formatting of the hintless 'show_ai_overlay=False' above clipped section

Reply:
lets add annotation then